redaxo/src/core/lib/rex.php

```diff
@@ -180,7 +201,19 @@ public static function getEnvironment()
      */
     public static function isDebugMode()
```

Comment:
Nachdem wir jetzt so viele modes haben, sollten wir vllt in den isXX methoden bissl beschreiben für was der mode alles zuständig ist